include/cpp-statsd-client/UDPSender.hpp

```diff
@@ -93,14 +99,14 @@ class UDPSender final {
     // @name Batching info
     // @{
 
-    //! Shall the sender use batching?
-    bool m_batching{false};
-
     //! The batching size
     uint64_t m_batchsize;
 
+    //! The sending frequency in milliseconds
+    uint64_t m_sendInterval;
+
     //! The queue batching the messages
-    mutable std::deque<std::string> m_batchingMessageQueue;
```

Comment:
i dont know why this was marked mutable, its in a non-const method with imho is properly non-const because queueing is a change to internal state that the caller should be aware of